public/src/feed/feed-mixin.html

```diff
@@ -99,6 +112,10 @@
         this.set("items", items);
       }
 
+      _appendItems(items) {
+        this.set('items', this.items.concat(items));
+      }
```

Comment:
DO we know if this will redraw ALL the card items? .push would add new cards at the bottom. 
Maybe polymer does the right thing here automatically. I don't know